_example/echo.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"net/http"
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+	"github.com/arl/statsviz"
+)
+
+func main(){
+	// Echo instance
+	e := echo.New()
+
+	// Middleware
+	e.Use(middleware.Logger())
+	e.Use(middleware.Recover())
+
+	// Routes
+	e.GET("/", hello)
+
+	// Start server
+	e.Logger.Fatal(e.Start(":1323"))
+
+	// Statsviz
+	// Add monitor Go application runtime statistics (GC, MemStats, etc.)
+	// Use http DefaultServeMux
+	mux := http.DefaultServeMux
```

Comment:
Can we create a `http.ServeMux` instead og using the default none here?

Reply:
Yes, we can create a `http.ServeMux`

Reply:
Ok then, can you modify the code so that `mux` is created with `http.NewServeMux` and not use the default.

Reply:
Ok, sure!

Reply:
I pushed new code!